tools/prepareLandscape.js

```diff
@@ -12,9 +12,28 @@ const items = require(path.resolve(projectPath, 'data.json'))
 const { website } = settings.global
 
 rmSync('public', { recursive: true, force: true })
-mkdirSync('public', { recursive: true })
-execSync(`cp -r "${projectPath}/images" public`)
```

Comment:
I'm assuming the issue you're having is with the `cp -r`. Maybe you could try using `fs.cp(src, dest, { recursive: true })`? See: https://nodejs.org/api/fs.html#fspromisescpsrc-dest-options

Reply:
Actually, `fs.cp` is async, so you would need to use the sync version: `fs.cpSync(src, dest, { recursive: true })`. See: https://nodejs.org/api/fs.html#fscpsyncsrc-dest-options

Reply:
ok.